plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryLargeImage.php

```diff
@@ -122,83 +117,42 @@ function ( $carry, $key ) use ( $directives ) {
 
 		ob_start();
 		?>
-			<ul class="wc-block-product-gallery-large-image__container" tabindex="-1">
-				<template data-wp-each--image="state.processedImageData" data-wp-each-key="context.image.id">
-					<?php // No need to use wp_kses on $directives_html because this markup is built internally. ?>
-					<?php // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
-					<li class="wc-block-product-gallery-large-image__wrapper" <?php echo $directives_html; ?>>
+			<ul
+				class="wc-block-product-gallery-large-image__container"
+				tabindex="-1"
+				data-wp-interactive="woocommerce/product-gallery"
+			>
+				<?php foreach ( $image_data as $image ) : ?>
+					<li class="wc-block-product-gallery-large-image__wrapper">
 						<img
 							class="<?php echo esc_attr( $base_classes ); ?>"
-							data-wp-bind--src="context.image.src"
-							data-wp-bind--srcset="context.image.srcset"
-							data-wp-bind--sizes="context.image.sizes"
-							data-wp-bind--data-image-id="context.image.id"
-							data-wp-bind--tabindex="context.image.tabIndex"
+							src="<?php echo esc_attr( $image['src'] ); ?>"
```

Comment:
To increase the LCP metric, what do you think if we increase the `fetchpriority` for the first image and lazy load the rest of the images?

```php
<?php if ( 0 === $index ) : ?>
	fetchpriority="high"
<?php else : ?>
	fetchpriority="low"
	loading="lazy"
<?php endif; ?>
```


| Before | After |
|--------|--------|
|<img width="1492" alt="image" src="https://github.com/user-attachments/assets/c9805133-dc01-4e7e-9519-61f77f224da6" />  | <img width="1493" alt="image" src="https://github.com/user-attachments/assets/e80f8f29-2254-4ed3-8a34-45c1fdded750" />|

Reply:
Amazing idea! Done https://github.com/woocommerce/woocommerce/pull/57519/commits/c2b21d030cfd0c30adecb18e8df8815b1ab28660